airflow/providers/google/cloud/hooks/bigquery.py

```diff
@@ -470,7 +470,7 @@ def create_empty_dataset(
 
         dataset: Dataset = Dataset.from_api_repr(dataset_reference)
         self.log.info('Creating dataset: %s in project: %s ', dataset.dataset_id, dataset.project)
-        self.get_client(location=location).create_dataset(dataset=dataset, exists_ok=exists_ok)
+        self.get_client(project_id=project_id, location=location).create_dataset(dataset=dataset, exists_ok=exists_ok)
```

Comment:
We should add a test to ensure that this is not removed in another PR

Reply:
Agree!